llms/tests/test_lora.py

```diff
@@ -45,7 +45,8 @@ def check_config(params):
                 v.size for _, v in tree_flatten(model.trainable_parameters())
             )
             self.assertEqual(
-                trainable_params, lora_layers * params["rank"] * 1024 * 2 * n_keys
+                trainable_params,
+                lora_layers * params["rank"] * 1024 * 2 * n_keys + lora_layers * n_keys,
```

Comment:
I couldn't find a way to make the scale a non-trainable constant parameter. Ideally, it shouldn't be part of trainable parameters since it's a constant parameter.